introduction.tex

```diff
@@ -13,9 +13,9 @@ \section{Welcome to the MEGA65!}
 Computer graphics and music make computing more fun; and we designed the MEGA65 for fun! In this user's guide, you will learn how to program using the MEGA65's built-in {\bf graphics} and {\bf sound} capabilities. But you don't need to be a programmer to have fun with the MEGA65. Because the MEGA65 includes a complete Commodore{\textregistered} 64{\texttrademark}\footnote{Commodore 64 is a trademark of C= Holdings}, it can also run thousands of games, utilities, and business software packages from the past, as well as new programs being written today by Commodore computer enthusiasts. Excitement for the MEGA65 will grow as we discover what programmers create as they learn about the power and features of this modern Commodore computer recreation. Together, we will build a new ``homebrew'' community to create software and projects we didn't think were possible on the MEGA65.
 
 We welcome you on this journey! Thank you for becoming a part of the {\bf MEGA65}
-community of users, programmers, and enthusiasts! Get involved, learn more
-about your MEGA65, and join us online at:
+community of users, programmers, and enthusiasts!
 
+% Get involved, learn more about your MEGA65, and join us online at:
 % TODO - I thought a call to action to join the community would be good to add early on. Where will the final online community call home?
```

Comment:
I wonder if the discord channel or the mega65-section of forum64.de would be worth a mention here:

- https://discord.gg/5DNvESf
- mega65.net  (redirects to forum64.de's mega65 section)

Reply:
I agree and advocate to mention the URLs.